NEWS.md

```diff
@@ -1,3 +1,10 @@
+## synapser 1.0.0
+___
+
+### Improvements
+* PythonEmbedInR dependency has been replaced by Reticulate ([SYNR-1310](https://sagebionetworks.jira.com/browse/SYNR-1310))
```

Comment:
Agreed. I was looking to conform with the format of previous NEWS entries which all referenced Jira tickets, but this is a special case, and it would be better to include the bullet points in the PR as well.

Reply:
Additional items from both the PR description, and the synapsePythonClient release notes for 2.5.0+ have been added to NEWS.rd

Reply:
Looks great!